README.md

```diff
@@ -56,11 +56,13 @@ PostgreSQL              | Ecto.Adapters.Postgres | [postgrex][postgrex]
 MySQL                   | Ecto.Adapters.MySQL    | [mariaex][mariaex]
 MSSQL                   | Tds.Ecto               | [tds_ecto][tds_ecto]
 SQLite3                 | Sqlite.Ecto            | [sqlite_ecto][sqlite_ecto]
+MongoDB                 | Mongo.Ecto             | [mongo_ecto][mongo_ecto]
```

Comment:
Please change the link name to "mongodb_ecto" since that is the application name.